src/StackAdmin/Azs.Subscriptions.Admin/Help/Move-AzsSubscription.md

```diff
@@ -0,0 +1,101 @@
+---
+external help file: Azs.Subscriptions.Admin-help.xml
+Module Name: Azs.Subscriptions.Admin
+online version:
+schema: 2.0.0
+---
+
+# Move-AzsSubscription
+
+## SYNOPSIS
+
+## SYNTAX
+
+```
+Move-AzsSubscription [[-DestinationDelegatedProviderOffer] <String>] [-ResourceId] <String[]> [-AsJob]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+Move subscriptions between delegated provider offers.
+This process will only perform a rebranding,
+the underlying offer, plans, quotas for the subscriptions will not be altered.
+
+## EXAMPLES
+
+### EXAMPLE 1
+```
+Move user subscriptions to a delegated provider offer.
```

Comment:
@caross this description should be below the code block it's in, and the PowerShell snippet below should be in this code block 😀 This comment applies to all examples added